### ISSUE TITLE
Error: Invalid Channel ID

### ISSUE THREAD

**Issue by CharlesARoy** [mfussenegger/nvim-dap-python#62]
This isn't a big deal for me as debugging seems to work fine for Python in spite of this error, but I just wanted to mention it in case it was indicative of a larger problem.

After running the DAP, when I exit nvim I see the following printed to the terminal:

```
...nvim/site/pack/packer/start/nvim-dap/lua/dap/session.lua:259: E900: Invalid channel id
stack traceback:
^I[C]: in function 'nvim_chan_send'
^I...nvim/site/pack/packer/start/nvim-dap/lua/dap/session.lua:259: in function <...nvim/site/pack/packer/start/nvim-dap/lua/dap/session.lua:258>%
```

My dap-python.lua config looks like:
```
local status_ok, dap_python = pcall(require, "dap-python")
if not status_ok then
  return
end

dap_python.setup('~/.virtualenvs/debugpy/bin/python')
```

And my dap.lua config looks like:

```
local dap_status_ok, dap = pcall(require, "dap")
if not dap_status_ok then
  return
end

local dap_ui_status_ok, dapui = pcall(require, "dapui")
if not dap_ui_status_ok then
  return
end

vim.fn.sign_define("DapBreakpoint", { text = "", texthl = "DiagnosticSignError", linehl = "", numhl = "" })

dapui.setup({
  icons = { expanded = "▾", collapsed = "▸" },
  mappings = {
    -- Use a table to apply multiple mappings
    expand = { "<CR>", "<2-LeftMouse>" },
    open = "o",
    remove = "d",
    edit = "e",
    repl = "r",
    toggle = "t",
  },
  -- Expand lines larger than the window
  -- Requires >= 0.7
  expand_lines = vim.fn.has("nvim-0.7"),
  -- Layouts define sections of the screen to place windows.
  -- The position can be "left", "right", "top" or "bottom".
  -- The size specifies the height/width depending on position. It can be an Int
  -- or a Float. Integer specifies height/width directly (i.e. 20 lines/columns) while
  -- Float value specifies percentage (i.e. 0.3 - 30% of available lines/columns)
  -- Elements are the elements shown in the layout (in order).
  -- Layouts are opened in order so that earlier layouts take priority in window sizing.
  layouts = {
    {
      elements = {
      -- Elements can be strings or table with id and size keys.
        { id = "scopes", size = 0.40 }, -- use 40% for scopes, evenly dividing the rest
        { id = "breakpoints", size = 0.20 },
        { id = "stacks", size = 0.20 },
        { id = "watches", size = 0.20 },
      },
      size = 60, -- 60 columns
      position = "left",
    },
    {
      elements = {
        "repl",
        -- "console",
      },
      size = 0.30, -- 30% of vertical lines
      position = "bottom",
    },
  },
  floating = {
    max_height = nil, -- These can be integers or a float between 0 and 1.
    max_width = nil, -- Floats will be treated as percentage of your screen.
    border = "single", -- Border style. Can be "single", "double" or "rounded"
    mappings = {
      close = { "q", "<Esc>" },
    },
  },
  windows = { indent = 1 },
  render = {
    max_type_length = nil, -- Can be integer or nil.
  }
})

dap.listeners.after.event_initialized["dapui_config"] = function()
  dapui.open()
end

dap.listeners.before.event_terminated["dapui_config"] = function()
  dapui.close()
end

dap.listeners.before.event_exited["dapui_config"] = function()
  dapui.close()
end
```

**Comment by mfussenegger** [comment]
Is your nvim-dap up2date? The messages shouldn't appear since https://github.com/mfussenegger/nvim-dap/commit/193e833dc0329e5ebcc37cf8a249c4e12c7332a6 which is included in both nvim-dap 0.1 and 0.2

**Comment by CharlesARoy** [comment]
Oh perfect, thank you. I was using nvim-dap with this commit:
`use { "mfussenegger/nvim-dap", commit = "014ebd53612cfd42ac8c131e6cec7c194572f21d" }`
